### DISCUSSION TITLE
code completion partially

### DISCUSSION THREAD

**Discussion by dss99911** [community/community#11596] [Copilot Conversations]
sometimes I would like to use only first some part of the code. for achieving that, for now, what I have to do is like the below

- Press the tab to input the suggestion
- remove not intended part.
but, if it's possible to add the part of the code by moving the cursor with some shortcut. it will be easier to use for the case.

**Comment by laylatichy** [comment]
+1 for this one, just got a tech preview a couple of hours ago and it's hard to believe that this suggestion is buried in a sea of "I can't install"

**Comment by Adrian-Samoticha** [comment]
This is something I have been thinking about as well. I've had Copilot generate huge pieces of code that oftentimes don't even compile, even though a part of the code happens to be precisely what I needed.

Accepting the code and removing the unwanted parts was oftentimes slower than typing out the wanted part myself, though.

**Comment by LegeBeker** [comment]
I would like this too. Maybe with `Option (⌥)` or `Alt`+`right` for each word, and `cmd (⌘)` or `ctrl`+`right` for a whole row.
